### PR TITLE
Fixes issue with opening conceptSet after relogin

### PR DESCRIPTION
**Pull request by wivern**
Fixes OHDSI/WebAPI#1595

### CODE REVIEW

File: js/components/welcome.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ define([
     'text!./welcome.html',
     'appConfig',
     'services/AuthAPI',
+	  'services/SourceAPI',
```
Comment (YaroslavTir) [inline js/components/welcome.js:6]: formating

File: js/components/welcome.js [hunk 2]
```diff
@@ -71,6 +73,9 @@ define([
 
         self.onLoginSuccessful = function(data, textStatus, jqXHR) {
             sharedState.resetCurrentDataSourceScope();
+            // if (!sharedState.vocabularyUrl()) {
+            //     await sourceApi.initSourcesConfig();
+            // }
```
Comment (YaroslavTir) [inline js/components/welcome.js:78]: :+1:

File: js/services/ConceptSet.js [hunk 3]
```diff
@@ -62,7 +62,8 @@ define(function (require, exports) {
 	}
 
 	function loadConceptSetExpression(conceptSetId) {
-		return httpService.doGet(config.api.url + 'conceptset/' + conceptSetId + '/expression/' + sharedState.sourceKeyOfVocabUrl()).then(({ data }) => data);
+		const sourceKey = sharedState.sourceKeyOfVocabUrl();
+		return httpService.doGet(config.api.url + 'conceptset/' + conceptSetId + '/expression' + (sourceKey ? `/${sourceKey}`: '')).then(({ data }) => data);
 	}
```
Comment (YaroslavTir) [inline js/services/ConceptSet.js:67]: this is used to be `/expression/` but now just '/expression'. Is this correct?

File: js/components/atlas-state.js [hunk 4]
```diff
@@ -8,14 +8,22 @@ define(['knockout', 'lscache', 'services/job/jobDetail', 'assets/ohdsi.util', 'c
...
+	state.defaultResultsUrl.subscribe((value) => state.resultsUrl(value));
 
 	state.resetCurrentDataSourceScope = function() {
-		state.resultsUrl(null);
-		state.vocabularyUrl(null);
-		state.evidenceUrl(null);
+		state.resultsUrl(state.defaultResultsUrl());
+		state.vocabularyUrl(state.defaultVocabularyUrl());
+		state.evidenceUrl(state.defaultEvidenceUrl());
 	}
```
Comment (YaroslavTir) [inline js/components/atlas-state.js:27]: can you keep the same order as above?
```
		state.vocabularyUrl(state.defaultVocabularyUrl());
		state.evidenceUrl(state.defaultEvidenceUrl());
		state.resultsUrl(state.defaultResultsUrl());
```

File: js/services/ConceptSet.js (hunk 3 above)
Comment (wivern) [reply js/services/ConceptSet.js:67]: It was resolved as `/expression/null` when `sourceKey` was `null`. And that's a mistake.
If talking regarding the tail slash, I should say the `/expression` and the `/expression/` has different meanings from HTTP's point of view.

**Pull request merged**